ManagedInjector.Lib/ManagedInjector.Lib.csproj

```diff
@@ -23,7 +23,10 @@
     </PropertyGroup>
 
     <ItemGroup>
+      <PackageReference Include="AsmResolver.DotNet" Version="4.5.1" />
+      <PackageReference Include="CommandLineParser" Version="2.8.0" />
       <PackageReference Include="Iced" Version="1.11.2" />
+      <PackageReference Include="MahApps.Metro" Version="2.4.5" />
```

Comment:
These package references are not needed, this is a library project that only handles injection.